sonar/modules/cli/utils.py

```diff
@@ -178,7 +185,89 @@ def export(pid_type, serializer_key, output_dir):
 
         click.secho('Finished', fg='green')
 
-    except Exception as exception:
-        click.secho('An error occured during export: {error}'.format(
-            error=str(exception)),
-                    fg='red')
+    except Exception as err:
+        click.secho(f'An error occured during export: {err}', fg='red')
+
+
+def create_personal(
```

Comment:
This can be a candidate for rero-invenio-base.

Reply:
Yes